docs/contexts/traceability-v1.jsonld

```diff
@@ -14,10 +14,10 @@
       "@id": "https://w3id.org/traceability/AgInspectionReport",
       "@context": {
         "facility": {
-          "@id": "https://w3id.org/traceability/#facility"
+          "@id": "https://www.gs1.org/voc/Place"
         },
         "inspector": {
-          "@id": "https://w3id.org/traceability/#inspector"
+          "@id": "https://www.lawinsider.com/dictionary/food-inspector"
```

Comment:
I think we need to break food inspector out to its own type if an appropriate one does not exist in GS1, else change reference to GS1

Reply:
I feel like [here](https://github.com/w3c-ccg/traceability-vocab/issues/53#issuecomment-743262231) @OR13 is saying to point to something human readable for explaining the property which seems to imply by referencing wikipedia that not having a schema backing it is okay.  BUT that being said, the link above is not my favorite either.  In fact, finding definitions of inspector was not easy.  So if we decided to break inspector out to its own type would we revert back to what I did before where we represent the definition that we're creating?    I'm good either way.